habitica.el

```diff
@@ -343,6 +349,31 @@ Options are 0.1, 1, 1.5, 2; eqivalent of Trivial, Easy, Medium, Hard."
   "Feed PET using FOOD."
   (habitica--send-request (format "/user/feed/%s/%s" pet food) "POST" ""))
 
+(defun habitica-api-get-groups (&rest types)
+  "Get groups which type is in TYPES."
+  (let* ((types (or types '("party" "guilds" "privateGuilds" "publicGuilds" "tavern")))
+         (types (string-join types ",")))
+    (message "types=%s" types)
+    (habitica--send-request (format "/groups?type=%s" (url-encode-url types)) "GET" "")))
+
+(defun habitica-api-get-group-id (&optional name)
+  "Get the id of group which name is NAME"
+  (let* ((groups (habitica-api-get-groups))
```

Comment:
might be worth caching those results if they aren't going to change frequently. but totally optional and can be a follow up.

Reply:
The result will be changing as you join or leave a group.